app/views/games/show.html.erb

```diff
@@ -9,25 +9,25 @@
   <div class="chessboard" id=<%= "#{@game.id}" %>>
     <% for row in (1..8) %>
       <% for col in (1..8) %>
-        <div class="square 
```

Comment:
I think we actually need the single space at the end of the line here on line 12, lines 17-19, and 26. Those are part of the spacing inside the `<div>` tag. (It's just being broken up over multiple lines for readability/looping.) Good eye for even noticing it though!

Reply:
Ah makes sense. I will fix it all up!